deploy/docs/Installation_with_Helm.md

```diff
@@ -78,7 +78,7 @@ helm upgrade --install my-release sumologic/sumologic --namespace=my-namespace -
 For Helm3, if the namespace does not exist, you can add the `--create-namespace` flag.
 
 ```bash
-helm upgrade --install my-release sumologic/sumologic --namespace=my-namespace --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME> --create-namespace"
+helm upgrade --install my-release sumologic/sumologic --namespace=my-namespace --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME>" --create-namespace
```

Comment:
I think we can add or strip a `"` for all of the external values

Reply:
Im fine with either approach TBH